crowdfunding/projects/serializers.py

```diff
@@ -0,0 +1,86 @@
+from rest_framework import serializers
+# for every model, create a serializer.
+# You can use a model serializer - like a model form.
+# Can build itself off a model. Automates
+
+from .models import Project, Pledge, Comment
+
+class PledgeSerializer(serializers.ModelSerializer):
+    '''
+    SlugRelatedField:
+    changing the way project title displays in GET request. Displays the title of the project and not the id # using slug (human readable label)
+
+    '''
+    project = serializers.SlugRelatedField(
+        queryset = Project.objects.all(),
+        slug_field='title'
+    )
```

Comment:
using title as a slug, bold step.
it should work since you made the field unique, but it might make life a little more difficult.
I would probably have considered creating an actual slug field on your model, which would have generated itself based on the title, just to give you a more sensible unique id.
really love that you've explored this though